odl/test/space/ntuples_test.py

```diff
@@ -1531,61 +1521,83 @@ def test_ufuncs(fn, ufunc):
         # Skip real only methods for complex
         return
 
-    # Get the ufunc from numpy as reference
-    npufunc = getattr(np, name)
-
     # Create some data
-    arrays, vectors = noise_elements(fn, n_args + n_out)
-    in_arrays = arrays[:n_args]
-    out_arrays = arrays[n_args:]
+    arrays, vectors = noise_elements(fn, nin + nout)
+    in_arrays = arrays[:nin]
+    out_arrays = arrays[nin:]
     data_vector = vectors[0]
-    in_vectors = vectors[1:n_args]
-    out_vectors = vectors[n_args:]
+    out_vectors = vectors[nin:]
+
+    if nout == 1:
+        out_arr_kwargs = {'out': out_arrays[0]}
+        out_vec_kwargs = {'out': out_vectors[0]}
+    elif nout == 2:
+        out_arr_kwargs = {'out1': out_arrays[0], 'out2': out_arrays[1]}
+        out_vec_kwargs = {'out1': out_vectors[0], 'out2': out_vectors[1]}
+    else:
+        assert False
+
+    # Get function to call, depending on the preferred interface
+    if USE_OLD_UFUNC_INTERFACE:
```

Comment:
Sure. I was getting annoyed by thousands of warnings from using the old interface :-)
Clearly I should find a way of printing the deprecation warning only once.